### DISCUSSION TITLE
How can I restart linter?

### DISCUSSION THREAD

**Discussion by Esnos33** [helix-editor/helix#7872] [Q&A]
Hi, do you know how can I restart linter? Currently I'm using pylint as linter, but the only way for me to have it update is to :wq and open file again. If there is some better linter then pylint that works faster in helix, I will be glad if you tell me about it.

**Comment by the-mikedavis** [comment]
You can restart a language server with `:lsp-restart`

**Reply by Esnos33** [reply]
I also find out, that manually saving also reloads pylint, I was always using auto save option.
